src/otaclient/_status_monitor.py

```diff
@@ -213,31 +229,44 @@ def _on_update_meta(status_storage: OTAClientStatus, payload: SetUpdateMetaRepor
             continue
         if v:
             setattr(update_meta, k, v)
+    return True
 
 
 #
 # ------ status monitor implementation ------ #
 #
 
 TERMINATE_SENTINEL = cast(StatusReport, object())
+MIN_COLLECT_INTERVAL = 0.5
+SHM_PUSH_INTERVAL = 0.5
 
 
 class OTAClientStatusCollector:
+    """NOTE: status_monitor should only be started once during whole otaclient lifecycle!"""
```

Comment:
Let me add one more detail: the OTAClientStatusCollector's life-cycle is the same as ota_core. It will only start once at the ota_core process starts, and keep running as a thread in the ota_core process.